test/e2e/lib/pages/plans-page.js

```diff
@@ -40,6 +40,26 @@ export default class PlansPage extends AsyncBaseContainer {
 		return await driverHelper.clickWhenClickable( this.driver, by.css( '.header-cake__back' ) );
 	}
 
+	async onePrimaryButtonShown() {
+		const count = await driverHelper.getElementCount(
+			this.driver,
+			by.css(
+				'.plan-features__table-item.is-top-buttons button.plan-features__actions-button.is-primary'
+			)
+		);
+
+		return count === 1;
+	}
+
+	async onePrimaryButtonShownForMobile() {
```

Comment:
It would be better to combine these in to one function and then do something like `if ( driverManager.currentScreenSize() === 'mobile' ) {` to  change the selector that it is getting the count of.

Reply:
Thanks @bsessions85 -- much cleaner!